resources/views/about/for-individuals.blade.php

```diff
@@ -79,7 +79,7 @@
             <x-section class="full accent" aria-labelledby="join">
                 <div class="center center:wide stack stack:xl">
                     <h2 class="text-center" id="join">{{ __('Join our accessibility community') }}</h2>
-                    <x-interpretation name="{{ __('Join our accessibility community', [], 'en') }}" namespace="join" />
+                    <x-interpretation name="{{ __('Join our accessibility community (individuals)', [], 'en') }}" />
```

Comment:
Might want to keep the name the as "Join our accessibility community" and just remove the namespace or change the namespace. Something that just occurred to me is that the localization strings won't be updated with unique names in this field. I think we should update the admin panel for the interpretations to show the namespace in the table view. We should probably have a chat about this when you're looking at things to work out the best approach.